pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -30,9 +32,7 @@ var (
 
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
-func (s Scanner) Keywords() []string {
-	return []string{"databricks", "dapi"}
-}
+func (s Scanner) Keywords() []string { return []string{"cloud.databricks.com", "dapi"} }
```

Comment:
Databricks has a number of related domains, including: `cloud.databricks.com`, `.azuredatabricks.net`, `gcp.databricks.com`, `dev.databricks.com`, and possibly more.

Reply:
Whoops I completely read the regex incorrectly. Thanks.